vortex-python/python/vortex/dataset.py

```diff
@@ -271,6 +282,7 @@ def take(  # pyright: ignore[reportIncompatibleMethodOverride]
         use_threads: bool | None = None,
         cache_metadata: bool = False,
         memory_pool: pyarrow.MemoryPool | None = None,
+        _row_range: tuple[int, int] | None = None,
```

Comment:
I wish there were a `RangeLike` typeclass so you could also take `range()`, list[int], etc.